doc/spec.md

```diff
@@ -1936,7 +1936,7 @@ argument it uses and how to convert it to a string.
 
 Each `%` character marks the start of a conversion specifier, unless
 it is immediately followed by another `%`, in which cases both
```

Comment:
Unrelated nit: s/cases/case/ on the line above.

Reply:
Thanks! Fixed.